src/app/clusters/ota-requestor/OTARequestorInterface.h

```diff
@@ -116,20 +116,15 @@ class ProviderLocationList
     /**
      * Delete a provider location from the list if found
```

Comment:
So down the line when other operations such as `DELETE` is supported, this would be useful for that. I guess for the time being, since we're not going to have support for individual deletes, this is fine.

nit: Please update the comment to reflect the change

Reply:
Ah, ok, but still, wouldn't "delete by fabric index" be enough?